packages/cli/lib/sandboxSync.js

```diff
@@ -157,53 +154,39 @@ const syncSandbox = async ({
           account: uiAccountDescription(accountId),
         })
       );
+    } else if (
+      isSpecifiedError(err, {
+        statusCode: 404,
+      })
+    ) {
+      uiCommandDisabledBanner(
```

Comment:
Just to clarify, this won't be throwing 404's yet right? This is just prepping the CLI to be able to gracefully handle them when it does?

Reply:
Yep exactly. The endpoint will end up sticking around until we go GA with v2 sandboxes which will be quite a while from now. The command should be removed long before the endpoint gets decommissioned so this may never get to run